site/src/api/errors.ts

```diff
@@ -115,18 +115,21 @@ export const getErrorDetail = (error: unknown): string | undefined => {
 		return error.detail;
 	}
 
-	if (error instanceof Error) {
-		return "Please check the developer console for more details.";
-	}
-
-	if (isApiError(error)) {
+	// APIErrors that are empty still benefit from checking the
+	// developer console if no detail is provided. So only use the
+	// detail field if it is not empty.
```

Comment:
this is the most mega nit thing ever, but these comments don't go out to 80 characters before wrapping 😂